Services/UI/OJS.Services.Ui.Business/Implementations/SubmissionsBusinessService.cs

```diff
@@ -254,22 +256,21 @@ public async Task<IEnumerable<SubmissionForProfileServiceModel>> GetForProfileBy
                 user.Id!,
                 isOfficial);
 
-        var validationResult = this.submissionResultsValidationService.GetValidationResult((user, problem, participant, isOfficial));
-        if (!validationResult.IsValid)
-        {
-            throw new BusinessServiceException(validationResult.Message);
-        }
+        return await this.GetUserSubmissions(problemId, isOfficial, take, user, problem, participant);
```

Comment:
Empty row here.

Its not a good practice to pass that much parameters just so you can pass them to the validation in the `GetUserSubmissions` method. This creates unnesessary dependencies. Better split your validation and use it in the `GetSubmissionDetailsResults` method.